geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegion.java

```diff
@@ -8075,8 +8075,12 @@ public DistributionAdvisee getParentAdvisee() {
 
     private final long maxTimeInRetry;
 
-    public RetryTimeKeeper(int maxTime) {
-      this.maxTimeInRetry = maxTime;
+    public RetryTimeKeeper(int maxTimeInRetry) {
+      this.maxTimeInRetry = maxTimeInRetry;
+    }
+
```

Comment:
why you need two methods here?

Reply:
It's not required. I added `public RetryTimeKeeper(long maxTimeInRetry)` because most millis are in long and the caller has to downcast with `(int)` which causes a warning in IntelliJ, and I was trying to reduce the number of warnings.